qcodes/instrument_drivers/stanford_research/SR830.py

```diff
@@ -154,6 +154,10 @@ class SR830(VisaInstrument):
     _N_TO_INPUT_CONFIG = {v: k for k, v in _INPUT_CONFIG_TO_N.items()}
 
     def __init__(self, name, address, **kwargs):
+
+        DeprecationWarning("This is an old driver an will be removed. Please use the the new driver: "
```

Comment:
DeprecationWarnings are hidden by default https://docs.python.org/3.6/library/warnings.html#default-warning-filters so we usually use a RunTimeWarbubg or UserWarning to make sure users actually see it

Reply:
A new kind of bug, the RunTimeWarBug! 😄

Reply:
What where I see nothing ;)